pixels-retina/pom.xml

```diff
@@ -87,6 +87,31 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>3.1.0</version>
```

Comment:
Define maven plugin version in the root pom. An example is `maven.plugin.source.version`.